app/components/forms/user-payment-info-form.js

```diff
@@ -0,0 +1,116 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { validPhoneNumber } from 'open-event-frontend/utils/validators';
+
+export default Component.extend(FormMixin, {
+
+  async init() {
+    this._super(...arguments);
+    let actualUser = await this.authManager.currentUser;
+    let userBillingInfo = {
```

Comment:
You can use `pick` from lodash for this.

Reply:
Done.